powdr_cli/src/main.rs

```diff
@@ -431,7 +431,14 @@ fn run_command(command: Commands) {
             (true, false) => {
                 let contents = fs::read_to_string(&file).unwrap();
                 let inputs = split_inputs::<GoldilocksField>(&inputs);
-                riscv_executor::execute::<GoldilocksField>(&contents, &inputs, &default_input());
+                let mut inputs_hash: HashMap<GoldilocksField, Vec<GoldilocksField>> =
+                    HashMap::default();
+                inputs_hash.insert(0u32.into(), inputs);
```

Comment:
I think it would be clearer to use `mapping` or something instead of `hash`, or just `inputs`.